test/integration/suites/k8s/conf/agent/spire-agent.yaml

```diff
@@ -78,6 +78,15 @@ data:
       }
     }
 
+    health_checks {
+      listener_enabled = true
+      bind_address = "0.0.0.0"
+      bind_port = "80"
```

Comment:
Can we use a different port other than 80? We're actively exploring ways to run SPIRE as a non-root user in containers and binding to port 80 will require elevated capabilities, so this will end up breaking.

Reply:
Ditto.